src/components/Buttontest.css

```diff
@@ -1,10 +1,11 @@
 .App-Button {
   position: absolute;
   top: 6vw;
-  right: 6vw;
-  height: 120px;
-  width: 340px;
+  right: 4vw;
+  height: 8vw;
+  width: 24vw;
```

Comment:
Your buttons currently don't scale well into mobile. I dunno if you care about having a mobile view, but if you do, then add some CSS media queries here to change the width, when the viewport goes above X amount.